SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -5,6 +5,7 @@
 #include "Engine/BlueprintGeneratedClass.h"
 #include "Engine/World.h"
 #include "UObject/TextProperty.h"
+#include "Net/NetworkProfiler.h"
```

Comment:
nit: please alphabetize includes

Reply:
@dbsigurd ^^

Reply:
Sorry about that, I had updated it with this note, but never realized that the build failed. Updated from release, and running the preflight now